### ISSUE TITLE
Resizing/ adjusting the width of the note pane changes also the width of the tags pane

### ISSUE THREAD

**Issue by elijahbrost** [standardnotes/web#377]
It should only resize the note (right), and note preview (middle) pane. Likewise, resizing the tags pane (left) should only change the width of the tags, and preview panes, and not that of the notes pane as well.

**Comment by baptiste-grob** [comment]
I agree the pane resizing behavior can be unpredictable. It's something we'll need to take a look at.
